request_a_govuk_domain/request/migrations/0011_alter_application_status_and_more.py

```diff
@@ -0,0 +1,407 @@
+# Generated by Django 4.2.13 on 2024-07-23 14:49
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
```

Comment:
Did we do that many changes to the model in this branch?

Reply:
Yes